src/handlers/watch-user-activity.ts

```diff
@@ -29,9 +29,9 @@ export async function watchUserActivity(context: ContextPlugin) {
     if (context.config.pullRequestRequired) {
       message.push(`- Be sure to link a pull-request before the first reminder to avoid disqualification.`);
     }
-    message.push(`- Reminders will be sent every \`${formatMillisecondsToDaysAndHours(context.config.warning / priorityValue)}\` if there is no activity.`);
+    message.push(`- Reminders will be sent every \`${formatMillisecondsToHumanReadable(context.config.warning / priorityValue)}\` if there is no activity.`);
     message.push(
-      `- Assignees will be disqualified after \`${formatMillisecondsToDaysAndHours(context.config.disqualification / priorityValue)}\` of inactivity.`
+      `- Assignees will be disqualified after \`${formatMillisecondsToHumanReadable(context.config.disqualification / priorityValue)}\` of inactivity.`
```

Comment:
What happens on priority 0?

Reply:
A disaster most likely, what should we do on priority 0 for the deadline?

Reply:
That or infinity. Let's try 1 for now then

Reply:
Even when the priority is `0` we still want the disqualifier to run right? If so I don't think infinity makes sense, it should use the same timers as a priority `1`.